src/babel/transformation/file/index.js

```diff
@@ -234,7 +222,46 @@ export default class File {
     stack = beforePlugins.concat(stack, afterPlugins);
 
     // register
-    this.transformerStack = stack.concat(secondaryStack);
+    this.transformerStack = this.mergeStack(stack.concat(secondaryStack));
+
+
+  }
+
+  mergeStack(_stack) {
+    var stack  = [];
+    var ignore = [];
+
+    for (let pass of (_stack: Array)) {
```

Comment:
[Yeah](https://github.com/babel/babel/blob/master/.babelrc#L5), I got sick of writing `for (var i = 0; i < arr.length; i++) {`.